packages/output/src/common/output-channel.ts

```diff
@@ -53,16 +75,16 @@ export class OutputChannelManager {
 
 export class OutputChannel {
 
-    private readonly visibilityChangeEmitter = new Emitter<{visible: boolean}>();
+    private readonly visibilityChangeEmitter = new Emitter<{ visible: boolean }>();
     private readonly contentChangeEmitter = new Emitter<OutputChannel>();
     private lines: string[] = [];
     private currentLine: string | undefined;
     private visible: boolean = true;
 
-    readonly onVisibilityChange: Event<{visible: boolean}> = this.visibilityChangeEmitter.event;
+    readonly onVisibilityChange: Event<{ visible: boolean }> = this.visibilityChangeEmitter.event;
     readonly onContentChange: Event<OutputChannel> = this.contentChangeEmitter.event;
 
-    constructor(readonly name: string, readonly preferences: OutputPreferences) { }
+    constructor(readonly name: string, readonly channelGroup: string, readonly preferences: OutputPreferences) { }
```

Comment:
please use options pattern to avoid breaking, i.e.
```
export interface OutputOptions {
    group?: string
}

constructor(readonly name: string, readonly preferences: OutputPreferences, options?: OutputOptions)
```